### ISSUE TITLE
Surrounding div 

### ISSUE THREAD

**Issue by uniboxx** [romkor/svelte-portal#183]
Hi, thanks for your work.
I would ask if there is a way to avoid the surrounding div of the ported element, of course in the case there is one main element that already contain all the elements of the main one.
Thanks in advance.

**Comment by RamIdeas** [comment]
I think it should have `style="display: contents;"` by default or a classname we can apply styles to it with

**Comment by phocks** [comment]
I agree. It would be good if we could remove it, or have a classname on it.
